.github/workflows/js_test.yml

```diff
@@ -0,0 +1,20 @@
+name: Javascript testing
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  eslint:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

Comment:
any reason for not using @v2 rather than @master ? (just reading the docs - https://github.com/actions/checkout - and seems like v2 is the one referenced)

Reply:
No... it looks like I used the configuration in https://github.com/marketplace/actions/yarn-cli but @v2 makes *much* more sense. I'll make the change.

Reply:
Oh... just noticed the deprecation warning on that

Reply:
Will switch to the suggested version.